src/client/.eslintrc.js

```diff
@@ -37,7 +37,6 @@ module.exports = {
 				extensions: ['.tsx', '.jsx'],
 			},
 		],
-		'sort-keys-fix/sort-keys-fix': ['error', 'asc', { caseSensitive: true, natural: true }],
```

Comment:
why remove sort-keys?

Reply:
Moves to top level eslint config because it's a common rule.